Sources/StructuredQueriesCore/TableAlias.swift

```diff
@@ -60,6 +60,21 @@ extension Table {
   /// // ON "users"."referrerID" = "referrers"."id"
   /// ```
   ///
+  /// Table aliases are representable in selections by providing the type to the `@Column` macro:
+  ///
+  /// ```swift
+  /// @Selection
+  /// struct UserWithReferrer {
+  ///   let user: User
+  ///   @Column(as: TableAlias<User, Referrer>.self)
```

Comment:
It seems that using the singular form doesn't work. The compiler complains that `User requires to conform to QueryBindable`. But using the plural forms `@Columns` works just fine.